datalad/cmd.py

```diff
@@ -325,13 +325,13 @@ def __call__(self,
                     except StopIteration:
                         # The process finished executing, store the last return
                         # code and restart the process.
-                        lgr.debug(f"{self}: command exited")
+                        lgr.debug("%s: command exited", self)
                         self.return_code = self.generator.return_code
                         self.runner = None
 
         except CommandError as command_error:
             # The command exited with a non-zero return code
-            lgr.error(f"{self}: command error: {command_error}")
+            lgr.error("%s: command error: command_error", self, command_error)
```

Comment:
dang, I did I screw up here, travis picked it up in https://app.travis-ci.com/github/datalad/datalad/jobs/577595841#L3864